### ISSUE TITLE
OMP Sentinels

### ISSUE THREAD

**Issue by hiker** [hiker/fab_new#1]
Some code contains:
```
!$ USE compute_chunk_size_mod, ONLY: compute_chunk_size  ! Note OpenMP sentinel
```
These dependencies are missed atm.

There is a work-around already merged into bom_master, but the next fparser release will provide support for omp conditional sentinels. Once this release is done, we only need to update the required fparser version, and provide the flag when parsing to enable handling of sentinels. THis will require the config object to 'know' if a build is using OpenMP or not.

**Comment by hiker** [comment]
Migrated to https://github.com/MetOffice/fab/issues/362
